vignettes/sir_models.Rmd

```diff
@@ -235,19 +235,16 @@ par(mfrow = c(2,4), oma=c(2,3,0,0))
 for (i in 1:N_age) {
   par(mar = c(3, 4, 2, 0.5))
   cols <- c(S = "#8c8cd9", I = "#cc0044", R = "#999966")
-  matplot(time, t(x[i,, ]), type = "l",
+  matplot(time, t(x[i+3,, ]), type = "l",
```

Comment:
Can you add a comment with an explanation of why the offset of 3 is needed in these functions